File: src/main/java/com/flexpoker/table/command/service/CardService.java
package com.flexpoker.table.command.service;

import java.util.List;

import com.flexpoker.table.command.Card;
import com.flexpoker.table.command.CardsUsedInHand;

public interface CardService {

    List<Card> createShuffledDeck();

    CardsUsedInHand createCardsUsedInHand(List<Card> fullDeckOfCards, int numberOfPlayers);

}

File: src/main/kotlin/com/flexpoker/table/command/service/CardService.kt
package com.flexpoker.table.command.service

import com.flexpoker.table.command.Card
import com.flexpoker.table.command.CardsUsedInHand

interface CardService {
    fun createShuffledDeck(): List<Card>
    fun createCardsUsedInHand(cards: List<Card>, numberOfPlayers: Int): CardsUsedInHand
}